packages/connection/src/class-manager.php

```diff
@@ -74,10 +74,17 @@ public function __construct( $plugin_slug = null ) {
 	public static function configure() {
 		$manager = new self();
 
+		add_filter(
+			'jetpack_constant_default_value',
+			__NAMESPACE__ . '\Utils::jetpack_api_constant_filter',
+			10,
+			2
+		);
+
 		$manager->setup_xmlrpc_handlers(
 			$_GET, // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 			$manager->is_active(),
-			$manager->verify_xml_rpc_signature()
+			class_exists( 'Jetpack' ) && $manager->verify_xml_rpc_signature()
```

Comment:
Thanks for this change @sergeymitr! My only concern is that we're disabling multiple xmlrpc methods, including some that don't seem to require the Jetpack plugin. For example `jetpack.getUser` doesn't seem to require Jetpack. I'm not sure whether any proposed consuming plugins plan to use any of those endpoints.

Have you had a chance to check each of the endpoints to make sure that the proposed consuming plugins aren't using them?

Reply:
@kbrown9 that's a really good point, I didn't think this through.
We should probably split the `jetpack.*` API methods into those that can function without Jetpack and those that can't.

This means that the quick fix didn't work out. I reverted my commit so you could merge, and will work on that in a separate PR.